scripts/test-eunomia.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -eu
+
+deploy_dev_image() {
+	echo "Deploying Dev Image Tag"
+	helm template deploy/helm/eunomia-operator/ | kubectl apply -f -
+	sleep 20
+	kubectl config set-context --current --namespace=eunomia-operator
+	helm template deploy/helm/eunomia-operator/ --set eunomia.operator.image.tag=dev | kubectl apply -f -
+	echo "Waiting for pod to come up"
+	sleep 30
+}
+
+deploy_latest_image() {
+	printf "\n\n********* Testing Latest image Tag *********\n"
+	helm template deploy/helm/eunomia-operator/ --set eunomia.operator.image.tag=latest | kubectl apply -f -
+	echo "Waiting for pod to come up"
+	sleep 30
+}
```

Comment:
why there are so many differences in deploy_latest_image vs. deploy_dev_image? I would expect they both could be nearly identical, what are the reasons they need to be so different?

Reply:
In deploy_dev_image  function
1) setting namespace  
2) applying all resources in  (deploy/helm/eunomia-operator/)

I believe those are not required  again while using latest tag ,so  ignored .

Reply:
ah; but then we could move those outside the 1st function, as a common preparation for both functions, and then the 2 functions could really be unified to just a 1 function with  1 parameter, no?

Reply:
Done with suggested changes .